ephios/templates/base.html

```diff
@@ -52,6 +52,7 @@
         {% endcompress %}
 
         {% compress js %}
+            <script type="text/javascript" src="{% static "vuejs/vue.global.js" %}"></script>
```

Comment:
sure that we want to serve vue for every view?

Reply:
it is cached and I thought about using vue in the signup configuration form, where you can't modify the included js.

Reply:
so it was a deliberate decision